docs/ads.md

```diff
@@ -0,0 +1,458 @@
+# Ads
+
+The Kaltura TV Platform Player supports the addition of 3rd party monetization solutions via its plugin system. To allow easy integration and to support playback and analytics using any 3rd party solution, the player defines an advertizing framework using APIs and a standard event scheme.
+
+Any plugin that adds support for ads is expected to follow the controller, timeline, and events scheme.
+This document details the ads controller, which allows controlling ad-related operations, the ad timeline (which explains how a typical order of events looks like), and the ad events and their associated payload.
+
+## Ads Controller
+
+To query the player regarding the ad's state in the current playback, you can use the ads controller by referencing `player.ads`.
+The ads controller is responsible for tracking and communicating with the active ads plugin, as well as for collecting and saving the current ads state and data.
```

Comment:
I'm not clear here if in "the current ads state and data" you mean a single ad or plural? In any case it should probably have a possessive ' added: "collecting and saving the current ad's state and data" or "collecting and saving the current ads' state and data".
Clear?

Reply:
@AnnaAleksandrowicz I believe it should describe any ad's state, which is currently being played.
I changed it to `To query the player regarding an ad's state in the current playback`